bridge/bridge.go

```diff
@@ -193,14 +193,17 @@ func (b *Bridge) add(containerId string, quiet bool) {
 		return
 	}
 
+	publishedPorts := 0
 	for _, port := range ports {
 		if b.config.Internal != true && port.HostPort == "" {
 			if !quiet {
 				log.Println("ignored:", container.ID[:12], "port", port.ExposedPort, "not published on host")
+			} else {
+				publishedPorts += 1
 			}
 			continue
 		}
-		service := b.newService(port, len(ports) > 1)
+		service := b.newService(port, publishedPorts > 1)
```

Comment:
This will not work for users that use the `-internal` flag, meaning they *do* want to use the exposed ports instead of the published ones.